Resources/Prototypes/Recipes/Construction/Graphs/structures/girder.yml

```diff
@@ -47,9 +59,9 @@
           conditions:
             - !type:EntityAnchored {}
           steps:
-            - material: Plasteel
+            - material: MetalRod
               amount: 2
-              doAfter: 1
+              doAfter: 5
```

Comment:
this will take years and actually buff RCDs indirectly, are you sure?

Reply:
oops our existing code is 4 seconds not 5